LogMonitor/src/LogMonitor/EtwMonitor.cpp

```diff
@@ -30,9 +30,11 @@ static const std::wstring g_sessionName = L"Log Monitor ETW Session";
 
 EtwMonitor::EtwMonitor(
     _In_ const std::vector<ETWProvider>& Providers,
-    _In_ std::wstring LogFormat
+    _In_ std::wstring LogFormat,
+    _In_ std::wstring CustomLogFormat

```

Comment:
I meant having `CustomLogFormat = L""` instead.

Reply:
@profnandaa even with making this change at the constructor level,  three variables will still need to be passed in the test cases.